blacksheep/server/normalization.py

```diff
@@ -301,6 +302,12 @@ def _get_parameter_binder(
     if isinstance(annotation, (str, ForwardRef)):  # pragma: no cover
         raise UnsupportedForwardRefInSignatureError(original_annotation)
 
+    if annotation is Request:
+        return RequestBinder()
+
+    if annotation is WebSocket:
+        return WebSocketBinder()
+
```

Comment:
That's a great one! Do I understand right that now the user can name the parameter that receives a WebSocket connection ot a Request object in the route function whatever they like? Moreover, they can put it as the second/third/etc. parameter in the function signature, right?

Reply:
Yes, you're correct! I must have missed this in the past, I thought that type-annotating a parameter with `: Request`  was already binding the request object, regardless of the name of the parameter. I think I made a mistake, apparently this wasn't supported.

As a side note, I know that my code in `normalization.py` is not elegant because it's hardcoding this kind of logic, but on the other hand I think it's user-friendly when defining request handler, and fast (normalizations are executed only once at application startup)

Reply:
Now that I think of it, maybe I will improve how aliases are handled and support extensions, to have a better solution for the next version of the framework. But let's keep this for another PR.